pkg/agent/manager/manager_test.go

```diff
@@ -169,13 +170,14 @@ func TestStoreSVIDOnStartup(t *testing.T) {
 
 	// Although start failed, the SVID should have been saved, because it should be
 	// one of the first thing the manager does at initialization.
-	svid, err := sto.LoadSVID()
+	svid, reattestable, err := sto.LoadSVID()
 	if err != nil {
 		t.Fatal(err)
 	}
```

Comment:
I think this can be condensed down to `require.NoError(t, err)`. Lots of this type of stuff in this file.